.github/workflows/kind.yml

```diff
@@ -60,6 +61,14 @@ jobs:
         make wait-for-work-agent
         make e2e-test-hosted-mode
 
+    - name: Test Coverage Verification
+      if: |
+        github.event_name == 'pull_request' &&
+        matrix.hosted_mode == 'false'
+      run: |
+        make test-coverage
+        make coverage-verify
+
```

Comment:
I suppose we'll add the Sonar step midstream? I've been thinking it'd be nice to clean that out from our repos upstream...

Reply:
Yes, looking at other repos, I thought that was the plan:
- [open-cluster-management-io/config-policy-controller coverage configuration](https://github.com/open-cluster-management-io/config-policy-controller/blob/main/.github/workflows/kind.yml#L88-L92)
- [stolostron/config-policy-controller coverage configuration](https://github.com/stolostron/config-policy-controller/blob/main/.github/workflows/kind.yml#L88-L113)